platform/commonUI/general/res/sass/mobile/_constants.scss

```diff
@@ -36,29 +36,29 @@ $mobileTreeRightArrowW: 30px;
 
 /************************** DEVICE WIDTHS */
 // IMPORTANT! Usage assumes that ranges are mutually exclusive and have no gaps
-$phoMaxW: 514px;
-$tabMinW: 515px;
-$tabMaxW: 1280px;
-$desktopMinW: 1281px;
+$phoMaxW: 767px;
+$tabMinW: 768px;
+$tabMaxW: 1024px;
```

Comment:
Previously the breakpoints were all contiguous. Now we have a no-man's land between tablet and desktop. Is this going to cause any problems?